### ISSUE TITLE
[Quest] A Fall From Grace

### ISSUE THREAD

**Issue by Shadowclaw95** [WoWManiaUK/Redemption#1618]
Greetings. I am currently on the quest "A Fall From Grace" in New Hearthglen (Dragonblight). What I am supposed to do, in order to complete the quest, is to ring the chapel bell and then to speak with the High Abbot. I do everything is requested and I follow him until we reach a cliff and then he throws himself down, as he is supposed to do. After all these actions I can read in the quest log that the quest is complete and that I have to return to Agent Skully in order to had in the quest. When I return to her though, the question mark above her head is still silver and I am not able to hand the quest in. It looks like I didn't complete the quest even if I do every task I am supposed to do. Regards, Lightrevenge.

http://www.wow-mania.com/armory/?quest=12274

**Comment by fvicaria** [comment]
![image](https://user-images.githubusercontent.com/9414661/30479283-76ae5050-9a0c-11e7-867f-a8ae22ac76a2.png)

**Comment by Shadowclaw95** [comment]
Yeah after he kills himselfs the quest is complete but it wasn't possible to me to hand the quest in to agent skully.

**Comment by captainloozer** [comment]
I've found a few weird things going on with the quest.  First, after talking with the high abbot and going to meet him by the cliff, I found him being attacked by some horses, which probably shouldn't happen.  Second, there were 3 of them for some odd reason, 2 jumped off the cliff, one stayed behind.  I got credit for him jumping.  Returned to Agent Skully and the question mark was still silver.  I abandoned the quest and had the same results.

Below is a video of my test.

https://www.youtube.com/watch?v=Z7DeGut8zUA

**Comment by fvicaria** [comment]
YOu need to stay close to the abbot within 5m.

**Comment by captainloozer** [comment]
Tried the quest again.  I still got two of the High Abbot npc's for some reason, and I stood right on top of both of them when they jumped.  The first one gave me credit but I still couldn't turn the quest in.  I recorded this attempt too:

https://youtu.be/ShZpUt-5Qa0

I even tried again after that, after there were no patrolling mobs to attack him, didn't have to fight or break my disguise. I stood right on top of him the whole time.  Got credit for the jump.  Still can't turn it in.
